pkg/dockerregistry/server/digestcache_test.go

```diff
@@ -0,0 +1,556 @@
+package server
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	"k8s.io/kubernetes/pkg/util/diff"
+)
+
+const (
+	allowedDeviation = time.Millisecond * 10
+
+	ttl1m = time.Minute
+	ttl5m = time.Minute * 5
+	ttl8m = time.Minute * 8
+)
+
+var stale = time.Unix(0, 0)
```

Comment:
> is this global var needed/used?

It is (https://github.com/openshift/origin/pull/9819/files#diff-76f17549cddaa8e6d3bfa366cebbc558R224) but I guess I could just leave stale entries with `evictOn` unset for the same result.